valhalla/thor/pathinfo.h

```diff
@@ -9,25 +9,29 @@ namespace valhalla {
 namespace thor {
 
 /**
- * Simple structure to pass path information from PathAlgorithm
+ * Simple(ish) structure to pass path information from PathAlgorithm
  * to TripLegBuilder
  */
 struct PathInfo {
-  sif::TravelMode mode;  // Travel mode along this edge
-  float elapsed_time;    // Elapsed time in seconds to the end of the edge
-  uint32_t trip_id;      // Trip Id (0 if not a transit edge).
-  baldr::GraphId edgeid; // Directed edge Id
-  float elapsed_cost;    // Cost to the end of the edge in cost units
-  bool has_time_restrictions;
+  sif::TravelMode mode; // Travel mode along this edge
+  float elapsed_time; // Elapsed time in seconds at the end of the edge including any turn cost at the
+                      // start of the edge
+  uint32_t trip_id;   // Trip Id (0 if not a transit edge).
+  baldr::GraphId edgeid;      // Directed edge Id
+  float elapsed_cost;         // Cost to the end of the edge in cost units
+  bool has_time_restrictions; // Whether or not this edge has a time restriction
+  float turn_cost; // Turn cost in seconds at the beginning of the edge, only in map matching for now
```

Comment:
formatting of comments sucks in the linter... anywya i added turn cost seconds here. we should have the other algorithms fill this out but for now its just map matching because it needs it to chop the match into legs and routes with the correct timing information.